src/models/spaces/organisms/organisms-row.ts

```diff
@@ -30,36 +30,52 @@ export const OrganismsRowModel = types
     hoveredOrganelle: types.maybe(Organelle),
     selectedOrganelle: types.maybe(Organelle),
     mode: types.optional(Mode, "normal"),
-    assayedOrganelle: types.maybe(Organelle),
+    assayedOrganelles: types.array(Organelle),
```

Comment:
I'm building up a simple array of assayed regions, but thinking back to a comment in the meeting yesterday do we need to think about ways to clear/reset this?